tests/ui/job-view/Push_test.jsx

```diff
@@ -49,21 +75,22 @@ describe('Push', () => {
       getApiUrl('/jobs/?push_id=511137', repoName),
       jobListFixture,
     );
-    fetchMock.get(
-      'https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.autoland.revision.d5b037941b0ebabcc9b843f24d926e9d65961087.taskgraph.decision/artifacts/public/manifests-by-task.json.gz',
-      404,
-    );
-    fetchMock.get(
-      'https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.autoland.revision.d5b037941b0ebabcc9b843f24d926e9d65961087.taskgraph.decision/artifacts/public/manifests-by-task.json',
-      {
-        'test-linux1804-64/debug-mochitest-devtools-chrome-e10s-5': [
-          'devtools/client/inspector/compatibility/test/browser/browser.ini',
-          'devtools/client/inspector/grids/test/browser.ini',
-          'devtools/client/inspector/rules/test/browser.ini',
-          'devtools/client/jsonview/test/browser.ini',
-        ],
-      },
-    );
+    const tcUrl =
+      'https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.autoland.revision.d5b037941b0ebabcc9b843f24d926e9d65961087.taskgraph.decision/artifacts/public';
+    // XXX: Fix this to re-enable test
+    // I need to figure out the right options to get a gzip blob
```

Comment:
The changes here don't actually do anything since the test is disable, however, I wanted *some* of my investigation not be lost so Ieft it here.

Reply:
Sounds good.  :)  I know just what you mean.